src/schemas/json/tslint.json

```diff
@@ -3937,6 +3937,26 @@
                         "additionalProperties": false
                       }
                     ]
+                  },
+                  "esSpecCompliant": {
+                    "description": "Option to forbid trailing comma after rest",
+                    "allOf": [
```

Comment:
@gitowiec, Not sure why allOf with `#/definitions/rule` is needed.
With this, the schema accepts files like:
```
{
  "rules": {
    "trailing-comma": {
      "options": {
        "esSpecCompliant": {
          "severity": "error"
        }
      }
    }
  }
}
```
Which doesn't seem to have any effect on TSLint.
If it's a simple boolean item [per the docs](https://palantir.github.io/tslint/rules/trailing-comma/), shouldn't simply
```
"esSpecCompliant": {
  "description": "Option to forbid trailing comma after rest",
  "type": "boolean"
}
```
be enough?